bad_code.java

```diff
@@ -45,9 +46,9 @@ public void doSomethingElse() {
         for (int i = 0; i < someNumber; i++) {
```

Comment:
Consider adding a break condition or modifying the loop to prevent potential performance issues with large values of `number`. [medium]